### PR TITLE
Use default string formatter for output

### LINKED ISSUE #9: Use stylelint's built-in formatters
Hello. Thanks for making a start on `grunt-stylelint`. It's a missing piece of the stylelint eco-system!

It's looking great so far. I have one suggestion though. I think you'll want to use stylelint's [built-in formatters](https://github.com/stylelint/stylelint/tree/master/src/formatters). The [default "string" formatter](https://github.com/stylelint/stylelint/blob/master/src/formatters/stringFormatter.js) is [quite powerful](https://cloud.githubusercontent.com/assets/808227/14970067/063e5802-10be-11e6-8774-6557d80c443b.png) and, I believe, the main reason someone would want to use a dedicated stylelint grunt plugin, rather than use the existing `grunt-postcss` and `postcss-reporter` combo.

Perhaps have a look at how [`grunt-eslint`](https://github.com/sindresorhus/grunt-eslint) passes [along its options](https://github.com/sindresorhus/grunt-eslint/blob/master/tasks/eslint.js#L27-L32) to eslint itself. You'll find stylelint's Node API options documented [here](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/node-api.md#options).

Thanks again and please don't hesitate to ask if you've any questions about the stylelint Node API :)

### PR DESCRIPTION
**Pull request by edg2s**
Fixes #9

### CODE REVIEW

File: tasks/grunt-stylelint.js [hunk 1]
```diff
@@ -13,23 +13,23 @@ module.exports = function ( grunt ) {
 		options.files = this.filesSrc.filter( function ( file ) {
 			return grunt.file.isFile( file );
 		} );
+		options.formatter = 'string';
 
 		styleLint.lint( options ).then( function ( data ) {
 			data.results.forEach( function ( result ) {
 				if ( !result.errored ) {
```
Comment (jeddy3) [inline tasks/grunt-stylelint.js]: The built-in ["verbose"](https://github.com/stylelint/stylelint/blob/master/src/formatters/verboseFormatter.js) formatter [display](https://cloud.githubusercontent.com/assets/808227/14970073/0ebc8bd4-10be-11e6-9398-4aafa0657fa4.png) a file-by-file status.

File: tasks/grunt-stylelint.js [hunk 2]
```diff
@@ -13,23 +13,23 @@ module.exports = function ( grunt ) {
...
 
+			if ( data.output ) {
+				if ( data.errored ) {
+					grunt.log.warn( data.output );
+				} else {
+					grunt.log.ok( data.output );
+				}
+			}
+
 			if ( !data.errored ) {
```
Comment (jeddy3) [inline tasks/grunt-stylelint.js]: As above, the "verbose" formatter also display this sucess info.

**Comment by jeddy3** [comment]
Looking good :) I've add two notes about the "verbose" formatter. If you go down the route of a [custom formatter](https://github.com/wikimedia/grunt-stylelint/issues/9#issuecomment-220435679) for the grunt formatting model, you can move these conditional into there too :)

File: tasks/grunt-stylelint.js (hunk 1 above)
Comment (edg2s) [inline tasks/grunt-stylelint.js]: We only want to show verbose output in the verbose log.

**Pull request merged**